docs/chymyst04.md

```diff
@@ -174,7 +174,7 @@ def makeCounter(initCount: Int): (M[Unit], B[Unit,Int]) = {
   val fetch = m[Unit, Int]
 
   join(
-    run { counter(n) + fetch(_, r) => counter(n) + r(n)},
+    run { counter(n) + fetch(_, r) => r(n)},
```

Comment:
I wanted `fetch` to fetch the value, not to also delete the counter.
I'd rather stay with the original code here.